Mashael_Almosallam/week_02/day_03/DOM-Manipulation/main.js

```diff
@@ -0,0 +1,74 @@
+
+//Change the font-family of the page to "Arial, sans-serif"
+document.body.style.fontFamily = "Arial, sans-serif";
+
+//Replace each of the span tags (nickname, favorite, hometown) with your own information (e.g. put your nickname in the nickname span)
+var newNickname = document.querySelector("#nickname");
+newNickname.innerHTML = ("Mashael");
+var newFavorite = document.querySelector("#favorite");
+newFavorite.innerHTML = ("Fish");
+var newHometown = document.querySelector("#hometown");
+newHometown.innerHTML = ("Riyadh");
+
+/* //Give each li the class of "listitem"
+var allListItems = document.querySelectorAll("li");
+var currentItems=allListItems.getAttribute = ("class", "listitem");
+//Change each li's text color to "rebeccapurple"
+currentItems.style.color = "rebeccapurple"; //NOT WORKING
+//debugger;
+ */
+
+var allListItems = document.querySelectorAll("li");
+
+for (var i = 0; i < allListItems.length; i += 1) {
+    var currentItem = allListItems[i];
+    currentItem.getAttribute = ("class", "listitem");
```

Comment:
So if you want to set a class for an element, you'd use `element.setAttribute("class", "class-name")`. What you did here will probably give you an error. 
[How to getAttribute()](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute)
[How to setAttribute()](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute)